R/Rarr_utils.R

```diff
@@ -0,0 +1,75 @@
+#' create_zarr_group
+#'
+#' create zarr groups
+#'
+#' @param store the location of (zarr) store
+#' @param name name of the group
+#' @param version zarr version
+#' @importFrom utils tail
+#' @examples
+#' store <- tempfile(fileext = ".zarr")
+#' create_zarr(store)
+#' create_zarr_group(store, "gp")
+#' @export
```

Comment:
Please be more consistent with the formatting of the function documentation, something like:

```suggestion
#' Create Zarr group
#'
#' Create a new group in a Zarr store
#'
#' @param store the location of the Zarr store
#' @param name name of the group to create
#' @param version Zarr version
#'
#' @noRd
#'
#' @examples
#' store <- tempfile(fileext = ".zarr")
#' create_zarr(store)
#' create_zarr_group(store, "gp")
```